### PR TITLE
Support ruby 3

### PR DESCRIPTION
**Pull request by dwilkie**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,10 +3,11 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'sinatra', require: nil
+gem 'ruby_speech', github: 'adhearsion/ruby_speech'
```
Comment (dwilkie) [inline Gemfile]: This can be removed if `ruby_speech` is bumped

Comment (bklang) [reply Gemfile]: This appears to be a new dependency, is it really necessary for Ruby 3 support?

Comment (dwilkie) [reply Gemfile]: It's a dependency in the gemspec already (https://github.com/adhearsion/adhearsion/blob/develop/adhearsion.gemspec#L42), but it looks like ruby_speech hasn't been bumped. (https://github.com/adhearsion/ruby_speech/compare/master..v3.0.0)

This commit (https://github.com/adhearsion/ruby_speech/commit/7c28b0302ee11ee55a8f0ea99f480a93ccc0fddb) allows later active support versions for `ruby_speech` but it's in the master branch. If you can bump the version of `ruby_speech` we can remove this line.

Comment (bklang) [reply Gemfile]: RubySpeech 3.0.1 has been published

Comment (dwilkie) [reply Gemfile]: Awesome. I removed this now.

**Review by bklang — APPROVED** [review]
LGTM, thanks!

**Comment by bklang** [comment]
In the absence of Travis, I ran these tests locally and all passed. Merging.

**Pull request merged**